spec/posix/1.0.md

```diff
@@ -7,26 +7,21 @@ The canonical URL for this document is <https://vt6.io/std/posix/1.0/>.
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
 
+Unless stated otherwise, this document implies all definitions of terms made by [`vt6/foundation`](https://vt6.io/std/foundation/) and other referenced modules.
```

Comment:
May this be a sentence `foundation` should require module specifications to always put at the beginning?

Reply:
I don't think it's necessary. This sentence is not there for the sake of `foundation`, but for the sake of the spec using it (`posix`, in this case).